css/main.css

```diff
@@ -0,0 +1,998 @@
+    font-weight: 800;
+    font-size: 17px;
+    line-height: 24px;
+    display: flex;
+    align-items: center;
+    text-align: center;
+    letter-spacing: 0.03em;
+    color: #6070ff;
+    align-self: flex-start;
+    padding: 20px 35px;
+    border-radius: 12px;
+    border: none;
+    margin: 7px auto;
+    width: 140px;
+  }
+
+  .reach_me button:hover {
+    background-color: #6070ff;
+    color: #fff;
+  }
+
+  .reach_me button:active {
+    background-color: #2230d2;
+    color: #fff;
+  }
+}
+
+.form-section {
+  background-color: #fff;
+  background-image:
+    url("../statics/images/ep1.svg"),
+    url("../statics/images/rp1.svg"),
+    url("../statics/images/ep2.svg"),
+    url("../statics/images/rp2.svg"),
+    url("../statics/images/ep3.svg"),
+    url("../statics/images/contact-form-\ background.svg");
+  background-repeat: no-repeat;
+  background-position:
+    78% 91%,
+    86% 90%,
+    99.7% 9.2%,
+    99.5% 6.2%,
+    41% 7.06%,
+    0;
+  background-size: auto, auto, auto, auto, auto, cover;
+  display: flex;
+  justify-content: center;
+  margin-top: 100px;
+}
+
+.div-wrap {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 100px 24px 0 24px;
+  text-align: center;
+}
+
+.form_t {
+  font-family: Poppins, sans-serif;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 40px;
+  line-height: 52px;
+  color: #fff;
+  margin-bottom: 20px;
+}
+
+.form-paragraph {
+  font-family: Poppins, sans-serif;
+  font-size: 20px;
+  line-height: 28px;
+  font-weight: 400;
+  text-align: center;
+  color: #ebebff;
+  margin: auto 15.5%;
+  padding: 0 14.5%;
+  font-style: normal;
+  margin-bottom: 50px;
+}
+
+.div-wrap .form-paragraph { margin-bottom: 24px; }
+
+.reach_me input {
+  font-family: Poppins, sans-serif;
+  font-size: 17px;
+  line-height: 20px;
+  display: flex;
+  align-items: center;
+  padding: 15px 10px;
+  border-radius: 12px;
+  border: none;
+}
+
+.reach_me textarea {
+  font-family: Poppins, sans-serif;
+  font-size: 15px;
+  line-height: 24px;
+  color: #b3bac5;
+  padding: 15px;
+  border-radius: 12px;
+  border: none;
+}
+
+.reach_me .button {
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 17px;
+  line-height: 24px;
+  display: flex;
+  align-items: center;
+  text-align: center;
+  letter-spacing: 0.03em;
+  color: #6070ff;
+  border: none;
+  width: 142px;
+  height: 48px;
+  padding: 12px 16px;
+  border-radius: 12px;
+  flex-direction: row;
+  justify-content: center;
+  margin: 24px auto;
+}
+
+@media (max-width: 720px) {
+  .reach_me {
+    flex-direction: column;
+    gap: 18px;
+    width: 90%;
+    margin: 24px 24px;
+  }
+
+  .div-wrap .form-paragraph {
+    font-family: Poppins, sans-serif;
+    font-size: 20px;
+    line-height: 28px;
+    font-weight: 400;
+    text-align: center;
+    color: #ebebff;
+    padding: 24px 1px;
+    margin: 0 5%;
+    font-style: normal;
+  }
+
+  .reach_me .button {
```

Comment:
[OPTIONAL] Please follow the Figma template and add hover and pressed states for your submit button like below.